src/components/Navigation/NavLink.component.tsx

```diff
@@ -1,31 +1,30 @@
-import styled from '@emotion/styled';
-import { Theme } from '@mui/material';
-import {
-  NavLink as NavLinkBase,
-  NavLinkProps as NavLinkBaseProps,
-} from 'react-router-dom';
+import styled from '@mui/material/styles/styled';
+import { Link as MuiLink } from '@mui/material';
+import { NavLink as NavLinkBase } from 'react-router-dom';
+import { LinkProps } from './NavLink.types';
 
-type NavLinkProps = NavLinkBaseProps & {
-  theme?: Theme;
-};
+const Link = (props: LinkProps) => (
+  <MuiLink component={NavLinkBase} {...props} />
+);
 
-export const NavLink = styled(NavLinkBase)(({ theme }: NavLinkProps) => ({
+export const NavLink = styled(Link)(({ theme }) => ({
```

Comment:
maybe let's move the styles to the `sx` prop? Just to make it more consistent with other parts of app

Reply:
Isn't it too long for sx?

Reply:
hmm, maybe that's true. So let's leave it like that to make it clearer